x-pack/plugins/uptime/common/runtime_types/monitor_management/monitor_types.ts

```diff
@@ -27,24 +28,34 @@ const Schedule = t.interface({
 export const TLSFieldsCodec = t.partial({
   [ConfigKey.TLS_CERTIFICATE_AUTHORITIES]: t.string,
   [ConfigKey.TLS_CERTIFICATE]: t.string,
-  [ConfigKey.TLS_KEY]: t.string,
-  [ConfigKey.TLS_KEY_PASSPHRASE]: t.string,
   [ConfigKey.TLS_VERIFICATION_MODE]: VerificationModeCodec,
   [ConfigKey.TLS_VERSION]: t.array(TLSVersionCodec),
 });
 
-export type TLSFields = t.TypeOf<typeof TLSFieldsCodec>;
+export const TLSSensitiveFieldsCodec = t.partial({
```

Comment:
If anyone knows how to recreate `Omit` in io.ts, please let me know because this implementation was a nightmare.